src/query/default_query_dispatcher.rs

```diff
@@ -14,7 +15,18 @@ use crate::shape::{HalfSpace, Segment, Shape, ShapeType};
 
 /// A dispatcher that exposes built-in queries
 #[derive(Debug, Clone)]
-pub struct DefaultQueryDispatcher;
+pub struct DefaultQueryDispatcher {
+    /// The absolute tolerance used by the GJK algorithm.
+    pub gjk_espilon_tolerance: Real,
```

Comment:
Create a `GjkOptions` struct ; implements default ; add `nb_max_iterations` ; check article gjk to add doc on that.